compiler/quilt/tools/command.py

```diff
@@ -394,6 +395,23 @@ def _check_team_login(team):
                 "Can't log in as a public user; log out from team %r first." % existing_team
             )
 
+def _check_team_exists(team):
+    """
+    Check that the team registry actually exists.
+    """
+    hostname = '%s-registry.team.quiltdata.com' % team
+    try:
+        socket.gethostbyname(hostname)
+    except IOError:
+        try:
+            # Do we have internet?
+            socket.gethostbyname('quiltdata.com')
+        except IOError:
+            message = "Can't find quiltdata.com. Check your internet connection."
+        else:
+            message = "Can't find the registry at %s. Make sure your team name is correct." % hostname
```

Comment:
At this point we know that only the `team_id` part of the registry is bad. So we should say something like:
`Unable to connect to registry. Is the team name {team_id} correct?`